src/fmu/dataio/dataio.py

```diff
@@ -761,8 +766,8 @@ def _establish_pwd_rootpath(self):
             if self._inside_rms or INSIDE_RMS or "RUN_DATAIO_EXAMPLES" in os.environ:
                 self._rootpath = (self._pwd / "../../.").absolute().resolve()
                 logger.info("Run from inside RMS (or pretend)")
-                self._inside_rms = True
-        # make some extra keys in settings:
+                # BUG(?): Should be ExportData._inside_rms ?
```

Comment:
@jcrivenaes 👀

Reply:
Yes a class variable initially; but I think the instance variable will just behave like (i.e. borrow from) the class variable?

Reply:
Im not sure, i have todo some RND. Will followup on this in another PR/issue.